<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dashboard</class>
 <widget class="QMainWindow" name="dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1516</width>
    <height>878</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="upload_btn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>203</y>
      <width>601</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_left_btn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rotate Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_right_btn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rotate Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoom_in_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>162</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zoom In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoom_out_btn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>162</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zoom Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>HOME PAGE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>171</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TODO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Company Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Company Logo</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>160</y>
      <width>159</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Document Details</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>88</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>PROCESSED</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>78</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>ARCHIVED</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>60</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>REPORT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>770</y>
      <width>75</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>SETTINGS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>210</y>
      <width>78</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Doc. Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>260</y>
      <width>100</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Doc. Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>310</y>
      <width>70</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>BP Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>360</y>
      <width>32</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>410</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Terms:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>470</y>
      <width>36</width>
      <height>19</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Code</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="doc_number">
    <property name="geometry">
     <rect>
      <x>992</x>
      <y>260</y>
      <width>491</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>20623</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bp_name">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>310</y>
      <width>491</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>SECONDS LOCKS REPAIR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="date">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>360</y>
      <width>491</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>JULY 07, 2024</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="terms">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>410</y>
      <width>491</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>CASH</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="description">
    <property name="geometry">
     <rect>
      <x>933</x>
      <y>510</y>
      <width>451</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fossil_fossil_Battery_replacement_one year warranty</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>470</y>
      <width>177</width>
      <height>19</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Description / QTY / Price</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>1366</x>
      <y>470</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Amount</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="amount">
    <property name="geometry">
     <rect>
      <x>1396</x>
      <y>510</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>84.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>560</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Total</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>610</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Discout:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>660</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Round-Off:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>710</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>GST:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>760</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>VAT:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>810</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Net Amount</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="round_off">
    <property name="geometry">
     <rect>
      <x>1396</x>
      <y>660</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>84.00</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gst">
    <property name="geometry">
     <rect>
      <x>1396</x>
      <y>710</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>84.00</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vat">
    <property name="geometry">
     <rect>
      <x>1396</x>
      <y>760</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>84.00</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="net_amount">
    <property name="geometry">
     <rect>
      <x>1396</x>
      <y>810</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>84.00</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="discount">
    <property name="geometry">
     <rect>
      <x>1396</x>
      <y>610</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>84.00</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="total">
    <property name="geometry">
     <rect>
      <x>1396</x>
      <y>560</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>84.00</string>
    </property>
   </widget>
   <widget class="QComboBox" name="doc_type">
    <property name="geometry">
     <rect>
      <x>992</x>
      <y>210</y>
      <width>491</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Sales Invoice</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sales Credit Note</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Purchase Invoice</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Purchase Debit Note</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Customer Receipt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vendor Payment</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="code">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>510</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>40110</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>40115</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50110</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50115</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>61000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>61200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>61300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>61400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>61500</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>61600</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="file_name">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>558</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FILE(Uploaded)-Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>130</y>
      <width>649</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Published</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>70</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>SEARCH BAR(BP Name)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
